roles/ssl-cert/files/ssl_cert.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# ssl_cert.sh DEST_DIR CERT_NAME CERN_CN
+#   DEST_DIR   - destination directory
+#   CERT_NAME  - base of filename of the key and csr file (eg. server)
+#   CERT_CN    - canonical name of the certificate for this key (default hostname)
+
+# ###########################################################################
+# mrg_ssl_create_key(CERT_NAME, CERN_CN)
+#   CERT_NAME  - base of filename of the key and csr file (eg. server)
+#   CERT_CN    - canonical name of the certificate for this key
+#
+#  Creates OpenSSL rsa keypair along wiht cert. signing request file (csr).
+# ###########################################################################
+function mrg_ssl_create_key() {
+  local CERT_NAME=$1
+  local CERT_CN=$2
+  test -f /etc/cron.d/redhat-ddns && \
+    local DDNS_NAME=$(head -1 /etc/cron.d/redhat-ddns | sed 's|.*name=\([-a-zA-Z0-9]*\)&.*|\1|').usersys.redhat.com
+  host ${CERT_CN} && local IP=$(host ${CERT_CN} | sed 's|[-0-9a-zA-Z.]* has address \([0-9.]*\)|\1|')
+
+
+  # Prepare openssl.cnf
+  :> /tmp/openssl.cnf
```

Comment:
please use ```mktemp``` with **RW** for user only

Reply:
@mkudlej addressed by 76a49f129e0f19517eb9aa9f597237b6fa996555

Reply:
The `openssl.cnf` file is in /tmp/ directory, because it might be worth to check how the certificate was configured. @mkudlej do you think, that I should create also the config file in the working (previously created temp directory), so it will be cleaned up together with the tmp dir?

Reply:
Well, if you think that you need to check this file because of debug in future, let it like it is.